components/charts/WordCloud.vue

```diff
@@ -13,39 +32,35 @@ export default {
     chartdata: new Chat(),
     minWordLength: {
       type: Number,
-      default: 3,
+      default: 3
     },
     minFontSize: {
       type: Number,
-      default: 6,
+      default: 6
     },
     randomness: {
       type: Number,
-      default: 0.1,
+      default: 0.1
     },
     stopWords: {
       type: Array,
-      default: () => stopwords,
-    },
+      default: () => stopwords
+    }
   },
   data() {
     return {
       chart: null,
       series: null,
+      additionalExcludes: ""
     };
   },
-  methods: {
-    updateGraph() {
-      this.chartdata.getAllWords().then((x) => (this.series.data = x));
-    },
-  },
   watch: {
     chartdata: {
       handler() {
         this.updateGraph();
       },
-      deep: true,
-    },
+      deep: true
+    }
```

Comment:
seems to be mainly formatting changes

Reply:
I don't know why, but the pipeline, was setup wrong; I fixed it and formatted all files again